docs/release-notes/8.4.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[release-notes-header-8.4.0]]
+== 8.4
+
+[discrete]
+[[release-notes-8.4.0]]
+=== 8.4.0
+
+[discrete]
+[[known-issue-8.4.0]]
+==== Known issue
+
+
+[discrete]
+[[breaking-changes-8.4.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+// end::breaking-changes[]
+
+[discrete]
+[[features-8.4.0]]
+==== Features
+* Creates a new rule type, New Terms, that fires when a value appears for the first time in a particular field ({pull}134526[#134526]).
+* Adds related alerts based on process ancestry to alert details ({pull}136009[#136009], {pull}138419[#138419], {pull}135340[#135340]).
+* Adds docs for Osquery API ({pull}137162[#137162]).
+* Creates the "Create dashboard" button to the Dashboards landing page ({pull}136671[#136671]).
+* Adds support for wildcard exceptions for detection rules. New operators are "matches" and "does not match" ({pull}136147[#136147]).
+* Adds a new search query parameter, `dry_run`, to the bulk actions API that allows you to simulate a bulk action without updating rules ({pull}134664[#134664]).
+* Creates the Responder, an interface that enables you to take actions on specific hosts ({pull}134520[#134520])
```

Comment:
@joepeeples flagging this in case you have a specific summary in mind for this feature.